tests/python_tests/utils/ov_genai_pipelines.py

```diff
@@ -49,7 +49,10 @@ def get_all_pipeline_types():
     return [PipelineType.STATEFUL, PipelineType.PAGED_ATTENTION, PipelineType.CONTINIOUS_BATCHING, PipelineType.SPECULATIVE_DECODING, PipelineType.PROMPT_LOOKUP_DECODING, PipelineType.AUTO]
 
 def get_main_pipeline_types():
-    return [ PipelineType.STATEFUL, PipelineType.PAGED_ATTENTION, PipelineType.SPECULATIVE_DECODING, PipelineType.PROMPT_LOOKUP_DECODING]
+    return [PipelineType.STATEFUL, PipelineType.PAGED_ATTENTION, PipelineType.SPECULATIVE_DECODING, PipelineType.PROMPT_LOOKUP_DECODING]
+
+def get_gguf_pipeline_types():
+    return [PipelineType.STATEFUL, PipelineType.PAGED_ATTENTION]
```

Comment:
after https://github.com/openvinotoolkit/openvino.genai/pull/1976 is merged, we will check whether GGUF model (modeling) can be converted to PA-based model.